src/components/filter/Filter.tsx

```diff
@@ -60,14 +60,17 @@ export const Filter = ({ filterTypes, initialFilters, setQueryVariables }: Props
   const [filters, setFilters] = useState(initialFilters);
   const [isCollapsed, setIsCollapsed] = useState(true);
 
-  useEffect(() => setQueryVariables(filters), [filters]);
+  useEffect(
+    () => setQueryVariables((prev) => ({ search: prev.search || '', ...filters })),
+    [filters]
```

Comment:
why is setting a default for search needed here? couldn't this be handled in Search component `setQueryVariables((prev) => ({ ...prev, search }));`?